scripts/setup.sh

```diff
@@ -42,7 +42,8 @@ as_user(){
 case "$ID" in
     rhel*|centos*)
         # Enable EPEL (required for Pip)
-        as_sudo 'yum -y install epel-release'
+        EPEL_URL="https://dl.fedoraproject.org/pub/epel/epel-release-latest-${VERSION_ID}.noarch.rpm"
+        as_sudo "yum -y install ${EPEL_URL}"
```

Comment:
This still didn't work. In the script that `VERSION_ID` appears as a decimal such as 7.9 or 8.3

Then it tries that url:
<https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.3.noarch.rpm>

But it doesn't exist. I modified the code to extract the first digit:
```
        version_=$(echo ${VERSION_ID} | sed  's/^[^0-9]*//;s/[^0-9].*$//')
        EPEL_URL="https://dl.fedoraproject.org/pub/epel/epel-release-latest-${version_}.noarch.rpm"
```

Then it goes to a valid url:
<https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm>

And the setup proceeds correctly.

Reply:
Thanks! Applied this fix.